core/pom.xml

```diff
@@ -40,6 +40,16 @@
       <classifier>${avro.mapred.classifier}</classifier>
     </dependency>
     <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-s3</artifactId>
+      <version>${aws.java.sdk.version}</version>
```

Comment:
1. shouldn't version declaration go into /pom.xml, with /core/pom.xml just declaring its use
1. Be aware that Hadoop branch-2 is on v. 1.10.6; HADOOP-12269. Between 1.7.4 and 1.10.6, Amazon did change the signature on one of the methods (int -> long on multipart upload). This means that s3a is very brittle about libraries, and some shading may be wise here. Or be in sync with the Hadoop 2.7 version and say "Don't use S3a on Hadoop 2.6.x", which is something the Hadoop team would concur with.